### DISCUSSION TITLE
Supports SQL-2023 Property Graph Queries

### DISCUSSION THREAD

**Discussion by xin-hao-2025** [vesoft-inc/nebula#5786] [Ideas]
See http://peter.eisentraut.org/blog/2023/04/04/sql-2023-is-finished-here-is-whats-new
The new SQL standard has introduced the `Property Graph Queries`, does Nebula have any plan to support this?

**Comment by QingZ11** [comment]
Nebula currently does not consider supporting SQL-2023 as we are focused on supporting GQL. Similar to SQL, GQL is a standardized query language for graph databases.

**Comment by xin-hao-2025** [comment]
@QingZ11 one more question
Is this the GQL? https://www.gqlstandards.org/home

**Reply by QingZ11** [reply]
You're spot on.
